### DISCUSSION TITLE
Error SetLogger Psr

### DISCUSSION THREAD

**Discussion by adityabayu1231** [mpdf/mpdf#1651] [Q&A]
i just installed laravel 9 and mpdf but i get error like this
Declaration of Mpdf\Mpdf::setLogger(Psr\Log\LoggerInterface $logger) must be compatible with Psr\Log\LoggerAwareInterface::setLogger(Psr\Log\LoggerInterface $logger): void
![Screenshot_20220423-230531_Telegram](https://user-images.githubusercontent.com/66105812/164913932-886d8592-8067-4e38-b56a-13fc2e0c1c23.jpg)
![Screenshot_20220423-230524_Telegram](https://user-images.githubusercontent.com/66105812/164913936-b70fd94b-8bd4-46bf-a76c-3f1bd9b295d5.jpg)

**Comment by finwe** [comment]
it seems you have installed `psr/log:^3.0`. mPDF is not compatible with it. Keep your dependencies at `^2.0`

**Reply by adityabayu1231** [reply]
Thank you
